cmd/rpctest/main.go

```diff
@@ -1262,6 +1305,65 @@ func bench6() {
 	}
 }
 
+func bench7() {
+	var client = &http.Client{
+		Timeout: time.Second * 600,
+	}
+	turbogethURL := "http://192.168.1.126:8545"
```

Comment:
we probably need to just move those to constants right now, they are repeating all over the rpc tests